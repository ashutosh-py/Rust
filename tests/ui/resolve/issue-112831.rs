// aux-build:issue-112831-aux.rs

mod zeroable {
    pub trait Zeroable {}
}

use zeroable::*;

mod pod {
    use super::*;
    pub trait Pod: Zeroable {} //~ ERROR expected trait, found derive macro `Zeroable`
}

use pod::*;

extern crate issue_112831_aux;
use issue_112831_aux::Zeroable;

fn main() {}
