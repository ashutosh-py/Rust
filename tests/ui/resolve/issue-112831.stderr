error[E0404]: expected trait, found derive macro `Zeroable`
  --> $DIR/issue-112831.rs:11:20
   |
LL |     pub trait Pod: Zeroable {}
   |                    ^^^^^^^^ not a trait
   |
help: consider importing this trait through its public re-export instead
   |
LL +     use Zeroable;
   |

error: aborting due to previous error

For more information about this error, try `rustc --explain E0404`.
